src/walk.rs

```diff
@@ -383,52 +377,39 @@ fn spawn_senders(
                 }
             };
 
-            if let Some(min_depth) = config.min_depth {
-                if entry.depth().map_or(true, |d| d < min_depth) {
-                    return ignore::WalkState::Continue;
+            let filters: Vec<Option<Box<dyn Filter>>> = vec![
```

Comment:
Instead of `Option` here, I'd prefer something like

```rust
if let Some(file_types) = config.file_types.clone() {
    filters.push(Box::new(file_types));
}
```

Reply:
I agree.